build/gulpfile.mixin.js

```diff
@@ -24,10 +24,17 @@ gulp.task('mixin', function () {
 		return;
 	}
 
-  // {{SQL CARBON EDIT}}
+	// {{SQL CARBON EDIT}}
+	let serviceUrl = 'https://raw.githubusercontent.com/Microsoft/sqlopsstudio/release/extensions/extensionsGallery.json';
```

Comment:
Is this always going to be the same value, or should it be configurable via environment variable?

Reply:
for now we only need one for release and one for insider 

---
In reply to: [174277667](https://github.com/Microsoft/sqlopsstudio/pull/902#discussion_r174277667) [](ancestors = 174277667)